inc/components/post-content/component.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Post content.
+ *
+ * Get the post concept.
+ *
+ * @todo Add support for template context.
+ * @todo Update the output to handle classic HTML and Gutenberg blocks.
+ *
+ * @package Irving_Components
+ */
+
+namespace WP_Irving;
+
+use WP_Irving\Component;
+
+if ( ! function_exists( '\WP_Irving\get_registry' ) ) {
+	return;
+}
+
+/**
+ * Register the component and callback.
+ */
+get_registry()->register_component_from_config(
+	__DIR__ . '/component',
+	[
+		'callback' => function( Component $component ): Component {
+
+			// Get the post ID from a context provider, or fallback to the global.
```

Comment:
Unnecessary since this is set in default context. Same for other post components below 🔢